assets/css/screen.css

```diff
@@ -375,39 +570,36 @@ The knock-on effect of this is ugly browser-scroll bars at the bottom, so 80px o
 /* 6. Post Feed
 /* ---------------------------------------------------------- */
 
+.posts {
```

Comment:
to be checked: each post card has a lightgrey bottom border (and they don't have any margins) in order to have vertical separation between the rows in the post list. the card list layout depends on many things (whether the post has image, the number of posts etc.) and I couldn't find a definite CSS rule to solve this in another way. 
this is a hack to hide the left and right side of these separator lines. unsure if this is okay / if there's any other way to solve it.

Reply:
sounds like a good one for @daviddarnes but might be helpful if you provide more context with screenshots